common/SmokeTests/SmokeTest/test-resources.json

```diff
@@ -277,9 +240,13 @@
             "type": "string",
             "value": "[reference(variables('keyVaultName')).vaultUri]"
         },
-        "EVENT_HUBS_CONNECTION_STRING": {
+        "EVENTHUB_NAMESPACE_CONNECTION_STRING": {
+            "type": "string",
+            "value": "[listkeys(variables('eventHubsAuthRuleResourceId'), '2015-08-01').primaryConnectionString]"
+        },
+        "EVENT_HUBS_TESTHUB_NAME": {
```

Comment:
Nit: why not a consistent "EVENTHUB_" prefix like the other above? Or, leave the other as it was.

Reply:
Good catch... that's a copy/paste mistake.   They should be consistent.